commit 488d4ed2949d8750610fb838d4283d188c9163dc
Author: Alexander Kurtakov <akurtako@redhat.com>
Date:   Mon Sep 22 14:42:04 2014 +0300

    Bug 444715 - Remove photon support from swt
    
    Remove remnants from photon support in javadoc and examples.
    
    Change-Id: I58e0adfac9f703abd96b9999d361b1c6e5f94ac6
    Signed-off-by: Alexander Kurtakov <akurtako@redhat.com>

1	1	bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/SWT.java
3	11	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/LinkTab.java
1	3	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
1	3	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
1	1	examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
1	1	examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
diff --git a/bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/SWT.java b/bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/SWT.java
index ac1ffc2..3719e3e 100644
--- a/bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/SWT.java	
+++ b/bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/SWT.java	
@@ -4300,7 +4300,7 @@ public static String getMessage(String key, Object[] args) {
 
 /**
  * Returns the SWT platform name.
- * Examples: "win32", "motif", "gtk", "photon", "carbon", "cocoa", "wpf"
+ * Examples: "win32", "gtk", "carbon", "cocoa", "wpf"
  *
  * @return the SWT platform name
  */
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/LinkTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/LinkTab.java
index 95e2773..c49dae3 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/LinkTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/controlexample/LinkTab.java
@@ -52,14 +52,8 @@ class LinkTab extends Tab {
 		if (borderButton.getSelection ()) style |= SWT.BORDER;
 	
 		/* Create the example widgets */		
-		try {
-			link1 = new Link (linkGroup, style);
-			link1.setText (ControlExample.getResourceString("LinkText"));
-		} catch (SWTError e) {
-			// temporary code for photon
-			Label label = new Label (linkGroup, SWT.CENTER | SWT.WRAP);
-			label.setText ("Link widget not suported");
-		}
+		link1 = new Link (linkGroup, style);
+		link1.setText (ControlExample.getResourceString("LinkText"));
 	}
 	
 	/**
@@ -79,9 +73,7 @@ class LinkTab extends Tab {
 	 */
 	@Override
 	Widget [] getExampleWidgets () {
-//		 temporary code for photon
-		if (link1 != null) return new Widget [] {link1};
-		return new Widget[] {};
+		return new Widget [] {link1};
 	}
 	
 	/**
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
index 99b6f03..1fa55f5 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/AlphaTab.java
@@ -229,9 +229,7 @@ public class AlphaTab extends AnimatedGraphicsTab {
 			return new Font(device, "Verdana", points, style);	
 		} else if (SWT.getPlatform() == "gtk") {
 			return new Font(device, "Baekmuk Batang", points, style);		
-		} else if (SWT.getPlatform() == "carbon") {
-			return new Font(device, "Verdana", points, style);
-		} else { // photon
+		} else { 
 			return new Font(device, "Verdana", points, style);
 		}
 	}
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
index 16d099b..07db1c0 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CountDownTab.java
@@ -260,7 +260,7 @@ public class CountDownTab extends AnimatedGraphicsTab {
 			return new String[] { "Courier", "Baekmuk Headline" }[index];
 		} else if (SWT.getPlatform() == "carbon") {
 			return new String[] { "Courier", "Impact" }[index];
-		} else { // photon, etc ...
+		} else { 
 			return new String[] { "Courier", "Verdana" }[index];
 		}
 	}
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
index 50ec191..33a5ff4 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesSWTTab.java
@@ -384,7 +384,7 @@ static String getPlatformFont() {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
 		return "Arial";
-	} else { // photon, etc ...
+	} else {
 		return "Verdana";
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
index 4b6f06f..d9157bc 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/CurvesTab.java
@@ -293,7 +293,7 @@ static String getPlatformFont() {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
 		return "Arial";
-	} else { // photon, etc ...
+	} else {
 		return "Verdana";
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
index 8855e21..3261c22 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/FontBounceTab.java
@@ -140,9 +140,7 @@ static String getPlatformFontFace(int index) {
 		return new String [] {"Arial", "Impact", "Times", "Verdana"} [index];	
 	} else if (SWT.getPlatform() == "gtk") {
 		return new String [] {"URW Chancery L", "Baekmuk Batang", "Baekmuk Headline", "KacsTitleL"} [index];
-	} else if (SWT.getPlatform() == "carbon") {
-		return new String [] {"Arial", "Impact", "Times", "Verdana"} [index];
-	} else { // photon, etc ...
+	} else { 
 		return new String [] {"Arial", "Impact", "Times", "Verdana"} [index];
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
index 7036eb7..9c85051 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/ImageFlipTab.java
@@ -105,7 +105,7 @@ static String getPlatformFont() {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
 		return "Arial";
-	} else { // photon, etc ...
+	} else {
 		return "Verdana";
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
index e00d369..0ea2652 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/InterpolationTab.java
@@ -192,7 +192,7 @@ static String getPlatformFont() {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
 		return "Arial";
-	} else { // photon, etc ...
+	} else {
 		return "Verdana";
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
index 23910bd..37801ad 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineCapTab.java
@@ -148,7 +148,7 @@ static String getPlatformFont() {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
 		return "Arial";
-	} else { // photon, etc ...
+	} else {
 		return "Verdana";
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
index efbf37f..7f4ff45 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/LineStyleTab.java
@@ -168,7 +168,7 @@ static String getPlatformFont() {
 		return "Baekmuk Batang";		
 	} else if (SWT.getPlatform() == "carbon") {
 		return "Arial";
-	} else { // photon, etc ...
+	} else {
 		return "Verdana";
 	}
 }
diff --git a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
index 4a49f41..11daa74 100644
--- a/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
+++ b/examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TextAntialiasTab.java
@@ -189,7 +189,7 @@ static String getPlatformFontFace(int index) {
 		return new String [] {"Luxi Mono", "KacstTitleL", "Baekmuk Batang", "Baekmuk Headline", "KacstFarsi", "Baekmuk Gulim", "URW Chancery L"} [index];
 	} else if (SWT.getPlatform() == "carbon") {
 		return new String [] {"Apple Chancery", "Brush Script MT", "Comic Sans MS", "Impact", "Herculanum", "Lucida Grande", "Papyrus"} [index];
-	} else { // photon, etc ...
+	} else {
 		return new String [] {"Courier", "Verdana", "Verdana", "Verdana", "Verdana", "Verdana", "Verdana"} [index];
 	}
 }
diff --git a/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java b/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
index 3499c31..575ed8f 100644
--- a/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
+++ b/examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/SnippetLauncher.java
@@ -57,7 +57,7 @@ public class SnippetLauncher {
 							platform = "JavaXPCOM";
 							skip = true;
 						} else {
-							String [] platforms = {"win32", "gtk", "photon", "carbon"};
+							String [] platforms = {"win32", "gtk", "carbon"};
 							for (int p = 0; p < platforms.length; p++) {
 								if (!platforms[p].equals(platform) && source.indexOf("." + platforms[p]) != -1) {
 									platform = platforms[p];
